Week3/sheepyis/Mission/src/apis/axios.tsx

```diff
@@ -0,0 +1,12 @@
+import axios from "axios";
+
+const baseURL = import.meta.env.VITE_API_BASE_URL;
+const acessToken = import.meta.env.VITE_API_KEY;
```

Comment:
이 부분에 오타가 있어 보입니다..!!

```suggestion
const accessToken = import.meta.env.VITE_API_KEY;
```